### PR TITLE
docs: improved baselining explanation

### PR DESCRIPTION
**Pull request by Jolg42**

### CODE REVIEW

File: content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx [hunk 1]
```diff
@@ -56,11 +56,15 @@ In other development environments (for example, other team members' computers):
 
 ## Baseline your production environment
 
-Assuming you have successfully transitioned to the new version of Prisma Migrate and initialized a new migration history in your repository, you now need to **baseline** any database that contains important data - such as production or staging.
+Baselining is for introducing Prisma Migrate to existing databases / brownfield projects. Concretely it's to tell Prisma Migrate to assume that one or more migration(s) were already executed so it won't try to drop and recreate existing tables.
 
-In development, we assume it is acceptable to reset your database to make sure your migration files are in sync with your database schema. This is not acceptable in a production environment, or any other environment where data must be maintained.
+Assuming you started a new migration history in your repository using `prisma migrate dev`, you now need to **baseline** any database that contains important data - such as production or staging.
```
Comment (2color) [inline content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx]:
```suggestion
Assuming you started a new migration history in your repository using `prisma migrate dev`, you now need to **baseline** any database that contains important data – such as production or staging – where the schema was not created using the `prisma migrate deploy` command.
```

**Review by 2color — APPROVED** [review]

File: content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx [hunk 2]
```diff
@@ -56,11 +56,15 @@ In other development environments (for example, other team members' computers):
 
 ## Baseline your production environment
 
-Assuming you have successfully transitioned to the new version of Prisma Migrate and initialized a new migration history in your repository, you now need to **baseline** any database that contains important data - such as production or staging.
+Baselining is for introducing Prisma Migrate to existing databases / brownfield projects. Concretely it's to tell Prisma Migrate to assume that one or more migration(s) were already executed so it won't try to drop and recreate existing tables.
```
Comment (tomhoule) [inline content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx]: Failing to baseline will not manifest in dropping and recreating tables, only failed migrations (the migration we would apply would try to create tables etc. that already exist).

File: content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx [hunk 3]
```diff
@@ -56,11 +56,15 @@ In other development environments (for example, other team members' computers):
...
 
-In development, we assume it is acceptable to reset your database to make sure your migration files are in sync with your database schema. This is not acceptable in a production environment, or any other environment where data must be maintained.
+Assuming you started a new migration history in your repository using `prisma migrate dev`, you now need to **baseline** any database that contains important data – such as production or staging – where the schema was not created using the `prisma migrate deploy` command.
```
Comment (tomhoule) [inline content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx]: I think "where the schema was not created using the `prisma migrate deploy` command" could be a bit confusing to users. Another way to formulate _when  you should baseline is "the first time you want to `prisma migrate deploy` to your existing database".

File: content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx [hunk 4]
```diff
@@ -56,11 +56,15 @@ In other development environments (for example, other team members' computers):
...
 
-Baselining works by creating the `_prisma_migrations` table (if it does not exist) and registering each baselined migration in the `_prisma_migrations` and as "already applied". Prisma Migrate will assume that the changes contained within "already applied" migrations already exist in the database, and will not run those migrations. Any migrations that come _after_ baselined migrations will actually be applied to the database.
+In development, we assume it is acceptable to reset and seed your database to make sure your migration files are in sync with your database schema. This is not acceptable in a production environment, or any other environment where data must be maintained.
+
+You will need to use `prisma migrate resolve --applied "MIGRATION_NAME"` (see example below) to register migration(s) in the `_prisma_migrations` as "already applied", the `_prisma_migrations` table will be created if it does not exist.
```
Comment (tomhoule) [inline content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx]: just for more context, we could maybe add "in the `_prisma_migrations` _table_"

File: content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx [hunk 5]
```diff
@@ -56,11 +56,15 @@ In other development environments (for example, other team members' computers):
...
+This will cause Prisma Migrate to ignore the "applied" migrations in your deployments.
+Any migrations that come _after_ baselined migrations will be applied to the database.
```
Comment (tomhoule) [inline content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx]:
```suggestion
When running `prisma migrate deploy`:
* Any "applied" migrations, whether actually applied or baselined (marked as applied) will be skipped.
* Any new migrations that come _after_ baselined migrations will be applied.
```

**Review by tomhoule — COMMENTED** [review]
nice!

File: content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx (hunk 3 above)
Comment (Jolg42) [reply content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx]: I agree with you :)

```suggestion
Assuming you started a new migration history in your repository using `prisma migrate dev`, you now need to **baseline** any database that contains important data – such as production or staging – the first time you want to `prisma migrate deploy` to your existing database.
```

File: content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx (hunk 4 above)
Comment (Jolg42) [reply content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx]:
```suggestion
You will need to use `prisma migrate resolve --applied "MIGRATION_NAME"` (see example below) to register migration(s) in the `_prisma_migrations` table as "already applied", this table will be created if it does not exist.
```

File: content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx (hunk 2 above)
Comment (Jolg42) [reply content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx]:
```suggestion
Baselining is for introducing Prisma Migrate to existing databases / brownfield projects. Concretely it's to tell Prisma Migrate to assume that one or more migration(s) were already executed so the generated migration(s) won't fail when they will try to create tables that already exist.
```

Comment (Jolg42) [reply content/300-guides/100-prisma-guides/110-add-prisma-migrate-to-a-project.mdx]: Better?

**Review by albertoperdomo — CHANGES_REQUESTED** [review]

**Comment by Jolg42** [comment]
I tried to merge your comments, does it makes sense now?

**Pull request merged**